### PR TITLE
DebugToolbar - Handle Query display in Query class

### PR DESCRIPTION
**Pull request by najdanovicivan**
**Description**
Moved Debug Toolbar Collector display method in Query class to allow overriding for custom highlighting in 3rd party DB drivers.

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPdocs
- [x] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: system/Database/Query.php [hunk 1]
```diff
@@ -473,6 +473,58 @@ protected function matchSimpleBinds(string $sql, array $binds, int $bindCount, i
 
 	//--------------------------------------------------------------------
 
+	/**
+	 * Returns string to display in debug toolbar
+	 *
+	 * @return string
+	 */
+	public function debugToolbarDisplay() : string
```
Comment (paulbalandan) [inline system/Database/Query.php:481]:
```suggestion
	public function debugToolbarDisplay(): string
```

File: system/Debug/Toolbar/Collectors/Database.php [hunk 2]
```diff
@@ -138,59 +138,12 @@ protected function formatTimelineData(): array
...
-			{
-				$sql = str_replace($term, "<strong>{$term}</strong>", $sql);
-			}
-
-			$data['queries'][] = [
-				'duration' => ($query->getDuration(5) * 1000) . ' ms',
-				'sql'      => $sql,
+		$data['queries'] = array_map(function ($query) {
+			return [
+				'duration' => (floatval($query->getDuration(5)) * 1000) . ' ms',
```
Comment (paulbalandan) [inline system/Debug/Toolbar/Collectors/Database.php:143]: let's modernize using typecasting
```suggestion
				'duration' => ((float) ($query->getDuration(5)) * 1000) . ' ms',
```

File: system/Debug/Toolbar/Collectors/Database.php [hunk 3]
```diff
@@ -138,59 +138,12 @@ protected function formatTimelineData(): array
...
-
-			foreach ($highlight as $term)
-			{
-				$sql = str_replace($term, "<strong>{$term}</strong>", $sql);
-			}
-
-			$data['queries'][] = [
-				'duration' => ($query->getDuration(5) * 1000) . ' ms',
-				'sql'      => $sql,
+		$data['queries'] = array_map(function ($query) {
```
Comment (paulbalandan) [inline system/Debug/Toolbar/Collectors/Database.php:141]: For IDE intellisense
```suggestion
		$data['queries'] = array_map(function (Query $query) {
```

Comment (najdanovicivan) [reply system/Debug/Toolbar/Collectors/Database.php]: Added 

I'm using PHPStorm and it already handles it properly as I changed 

```
	 * @var Query[]
	 */
	protected static $queries = [];
```

So I missed adding it in closure definition

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**